css/style.css

```diff
@@ -738,8 +741,20 @@ button.button-inline:hover {
 
 #boardlist td .app-popover-menu-utils {
 	float: right;
+	width: 30px;
+	display: inline;
+	position: relative;
+}
+
+#boardlist td .app-popover-menu-utils button {
+	opacity: 0.3;
 }
 
+#boardlist td .app-popover-menu-utils:hover button {
+	opacity: 0.7;
```

Comment:
That doesn't make it correct, @artemanufrij - plus it gives us a third opacity behaviour for the same button:
- triple-dot in navigation bar: 0.5->1
- triple-dot on card: 0.25->1
- triple-dot on board list: 0.3->0.7

I will make a PR to Files to fix it there as well.